Python/setup.py

```diff
@@ -120,7 +120,7 @@ def finalize_options(self):
         self.define = self.define or []
         self.libraries = self.libraries or []
 
-        extra_compile_args = []
+        extra_compile_args = ['-O2', '-stdlib=libc++', '-mmacosx-version-min=10.9']
```

Comment:
This shouldn't be here.  If you need these extra arguments, you can pass them by setting the `CXXFLAGS` environment variable and they will be picked up by the setup script.